### ISSUE TITLE
Editor changes in 1.9.0-beta.10?

### ISSUE THREAD

**Issue by zegab1** [notable/notable#1660]
I've noticed that in 1.8.4, triple quotes (```) automatically completed with a closing pair. This was extremely helpful if I needed to insert a lot of code snippets with copy-paste. 1.9.0-beta.10 do the same for double quotes (was not done in 1.8.4), but doesn't complements triple quotes any more. Brackets (){}[] are complemented in both versions. 

- Is this an expected behaviour?
- Is this configurable somewhere?
- Is there any documantation what is complemented in the editor?
- can we get aut-complete for triple quotes back in 1.9.x?

**Comment by fabiospampinato** [comment]
> Is this an expected behaviour?

Yes, however different people want this to work differently, this should be configurable really.

> Is this configurable somewhere?

Not yet.

> Is there any documantation what is complemented in the editor?

The following characters are autocompleted: "([{<

> can we get aut-complete for triple quotes back in 1.9.x?

Maybe, I need to figure out a nice way to make this customizable. For now there are more pressing issues.
